docs/pages/installation/codespaces.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: default
+title: Codespaces
+permalink: installation/platforms/codespaces
+type: installation
+display-title: "false"
+language: en
+list: include
+image: /assets/img/platforms/codespaces.png
+---
+
+
+
+<h1>Quick Start with {{ page.title }} <img src="{{ page.image }}" style="width:35px;height:35px;" /></h1>
+
+We will use Minikube in Codespaces to setup developer environment for Meshery.
+
+**To Setup and run Meshery on Minikube** :
+
+- [Steps](#steps)
+  - [1. Start minikube](#1-start-minikube)
+  - [2. Install Meshery](#2-install-meshery)
+  - [3. Configure Meshery to use minikube](#3-configure-meshery-to-use-minikube)
+  - [Manual Steps](#manual-steps)
+
+## Steps
+
```

Comment:
Before that, one suggestion we can include "Deploy CodeSpaces" button here.... Refer here... https://docs.github.com/en/codespaces/setting-up-your-project-for-codespaces/setting-up-your-repository/adding-a-codespaces-badge

Reply:
Hi, thank you for your suggestion. I have updated the docs with a Codespace badge to open the repo with a single click.

![codespace_badge](https://user-images.githubusercontent.com/97174641/218274876-c6f85ae1-4beb-41f7-bc51-9c062fcda14b.png)